app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/presentation/common/ViewExt.kt

```diff
@@ -0,0 +1,51 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.presentation.common
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Parcelable
+import android.text.Editable
+import android.text.Html
+import android.text.TextWatcher
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Toast
+import com.bumptech.glide.Glide
+
+fun ImageView.loadImage(photoUrl: String) {
+    Glide.with(context)
+            .load(photoUrl)
+            .into(this)
+}
+
+fun ImageView.loadImageCenterCropped(photoUrl: String) {
```

Comment:
you can call `fun ImageView.loadImage(photoUrl: String, cropped: Boolean = false)`